scripts/common.ts

```diff
@@ -7,6 +7,27 @@ import * as LocalConfig from '../deployment.json';
 
 /* Utility functions */
 
+export function artifacts(path: string): {[key: string]: {[key: string]: {"address": string, "constructor_args": any[]}}} {
+  let ARTIFACTS: {[key: string]: {[key: string]: {"address": string, "constructor_args": any[]}}} = LocalConfig.ARTIFACTS;
+  return ARTIFACTS;
+}
+
+export function artifact_addr(path: string): string|null {
+  let ARTIFACTS: {[key: string]: {[key: string]: {"address": string, "constructor_args": any[]}}} = LocalConfig.ARTIFACTS;
+  if (path in ARTIFACTS) {
+    return ARTIFACTS[path]["address"];
+  }
+  return null
+}
+
+export function artifact_constructor_args(path: string): any[] {
```

Comment:
can't see where this function is being used.
If not used, do we need it for now?